src/Model/TransportModel/tsp1apt1.f90

```diff
@@ -1609,6 +1615,22 @@ subroutine apt_read_cvs(this)
         ! -- strt
         this%strt(n) = this%parser%GetDouble()
         !
+        ! -- If GWE model, read additional thermal conductivity terms
```

Comment:
Should remove this in the future.  Generic transport routine should not need to know about temperature.

Reply:
@langevin-usgs Agreed, this should be backed out.  I tried backing out, but didn't know where to put it as SFE, LKE, and MWE all need it.  SFT, LKT, MWT, UZT, & UZE do not.  Is the only way to overcome this to add a copy of the function to SFE, LKE, and MWE with the additional lines added so that we can override the function in APT?  It'll add a fair bit of duplication, but it certainly is cleaner.